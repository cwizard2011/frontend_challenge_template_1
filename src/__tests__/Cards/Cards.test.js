import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Card from '../../Cards';

describe('<Cards />', () => {
  const props = {
    thumbnail: 'alsace.gif'
  }
  it('renders without crashing', () => {
   const wrapper = shallow(<Card {...props} />);
   expect(toJSON(wrapper)).toMatchSnapshot();
  })
})