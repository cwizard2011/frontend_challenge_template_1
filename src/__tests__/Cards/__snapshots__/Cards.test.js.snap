// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cards /> renders without crashing 1`] = `
<div
  onClick={[Function]}
  style={
    Object {
      "borderRadius": 5,
      "boxShadow": "0.5rem 0.5rem 3rem rgba(0,0,0,0.2)",
      "boxSizing": "border-box",
      "cursor": "pointer",
      "marginBottom": 15,
      "marginLeft": 30,
      "marginTop": 50,
      "overflow": "hidden",
      "width": 200,
    }
  }
>
  <Card
    tag="div"
  >
    <CardImg
      alt="Card image cap"
      src="alsace.gif"
      tag="img"
      top={true}
      width="100%"
    />
    <CardBody
      tag="div"
    >
      <CardTitle
        style={
          Object {
            "fontFamily": "Montserrat",
            "fontWeight": "bold",
            "textAlign": "center",
          }
        }
        tag="div"
      />
      <div
        style={
          Object {
            "display": "flex",
            "justifyContent": "space-between",
          }
        }
      >
        <CardSubtitle
          style={
            Object {
              "fontFamily": "Montserrat",
              "marginLeft": 15,
              "textDecoration": "line-through",
            }
          }
          tag="div"
        >
          $
        </CardSubtitle>
        <CardSubtitle
          style={
            Object {
              "fontFamily": "Montserrat",
            }
          }
          tag="div"
        >
          $
        </CardSubtitle>
      </div>
    </CardBody>
  </Card>
</div>
`;
